[INFO] [launch]: All log files can be found below /root/.ros/log/2025-03-04-13-44-26-399807-drone-home-2554
[INFO] [launch]: Default logging verbosity is set to INFO
[INFO] [cv_controller-1]: process started with pid [2566]
[INFO] [controller-2]: process started with pid [2568]
[INFO] [pwm_publisher-3]: process started with pid [2570]
[INFO] [gps_publisher-4]: process started with pid [2572]
[INFO] [imu_publisher-5]: process started with pid [2574]
[INFO] [drive_hardware-6]: process started with pid [2576]
[cv_controller-1] [INFO] [1741113871.806561512] [drone_home1.sim]: CV Controller running
[controller-2] [INFO] [1741113872.571517355] [drone_home1.controller]: Controller running
[imu_publisher-5] [INFO] [1741113873.985315713] [drone_home1.imu_publisher]: Calibration status: (True, 'Calibration (sys:0, gyro:0, accel:3, mag:0)')
[drive_hardware-6] [INFO] [1741113879.933506624] [drone_home1.drive_hardware]: Service request to change drive mode custom_messages.srv.SetMode_Request(mode='manual')
[drive_hardware-6] [INFO] [1741113879.939195741] [drone_home1.drive_hardware]: Drive mode now: manual
[controller-2] [INFO] [1741114101.892143378] [drone_home1.controller]: Service request to change coordinate custom_messages.srv.SetCoordinate_Request(new_coordinate=sensor_msgs.msg.NavSatFix(header=std_msgs.msg.Header(stamp=builtin_interfaces.msg.Time(sec=0, nanosec=0), frame_id=''), status=sensor_msgs.msg.NavSatStatus(status=0, service=0), latitude=29.64736, longitude=-82.34719, altitude=0.0, position_covariance=array([0., 0., 0., 0., 0., 0., 0., 0., 0.]), position_covariance_type=0))
[controller-2] [INFO] [1741114101.897638852] [drone_home1.controller]: Waypoints now: [(29.64736, -82.34719)]
